### DISCUSSION TITLE
Default alert message template

### DISCUSSION THREAD

**Discussion by bobemoe** [grafana/grafana#46329] [Alerting]
# Use case

I'm wanting to customise the alert email message. Just want to make some small tweaks,

# Backgroud
By default the contact points and message templates are empty. Granted there are links to templating and variable docs.

I create a template but have to search how to link the template to a contact point. Seems I need to edit the message and put a link to the template in there. Still not quite figured that out yet. 

As for the email template, I'll have to start crafting from scratch referencing docs for variable names. (this will help https://github.com/grafana/grafana/issues/35726)

# Feature Request
If would be great if the default email template was already available in the templates list for editing or duplication. This would give me a much better starting point for customisation.

**Comment by gelicia** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

This discussion was converted into an issue, which is available [here](https://github.com/grafana/grafana/issues/73446).
